ObjectiveCloudant/Operations/CDTQueryViewOperation.h

```diff
@@ -0,0 +1,193 @@
+//
+//  CDTQueryViewOperation.h
+//  ObjectiveCloudant
+//
+//  Created by Rhys Short on 15/02/2016.
+//  Copyright (c) 2016 IBM Corp.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+//  except in compliance with the License. You may obtain a copy of the License at
+//    http://www.apache.org/licenses/LICENSE-2.0
+//  Unless required by applicable law or agreed to in writing, software distributed under the
+//  License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+//  either express or implied. See the License for the specific language governing permissions
+//  and limitations under the License.
+//
+
+#import <ObjectiveCloudant/ObjectiveCloudant.h>
+
+typedef NS_ENUM(NSUInteger, CDTStale) {
+    /**
+     Do not allow stale views.
+     **/
+    CDTStaleViewNo,
+    /**
+     Allow stale views.
+     **/
+    CDTStaleViewOk,
+    /**
+     Allow stale views, but update them immediately after the request.
+     **/
+    CDTStaleViewUpdateAfter
+};
+
+/**
+ An operation to query Map Reduce views.
```

Comment:
Also this operation doesn't support the (at least in Cloudant) option to POST multiple queries to a view.